lib/src/bezier.dart

```diff
@@ -725,4 +725,66 @@ abstract class Bezier {
 
     return lookUpTable;
   }
+
+  /// Returns a [ProjectionResult] object with the closest point to the closest off curve point 
+  /// provided as the [point] parameter.
+  ///
+  /// Finds the on-curve point closest to the specific off-curve point, using a two-pass projection 
+  /// test based on the curve's LUT. A distance comparison finds the closest match, after which a 
+  /// fine interval around that match is checked to see if a better projection can be found.
+  ProjectionResult project(Vector2 point) {
+    // coarse check
+    final List<Vector2> lookUpTable = positionLookUpTable(intervalsCount: 100);
+
+    // Find closest in LUT
+    double mdist = double.maxFinite;
+    int mpos = 0;
+    int length = lookUpTable.length;
+    for (int i = 0; i < length; i++) {
+      final Vector2 p = lookUpTable[i];
+      double d2 = p.distanceToSquared(point);
```

Comment:
Good idea to use the squared distance here!  💡👍 It doesn't look like Bezier.js is doing this, but I think they could benefit from it too.  ➰